### PR TITLE
Polls 2022-04-04

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

File: governance/polls/Maker Wormhole Deployment - April 4, 2022.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## Review
+
+The community can vote in this poll to express support or opposition to:
+* Launching Maker Wormhole
+* Approving the Gateway contract for the Arbitrum Domain
+* Approving the Gateway contract for the Optimism Domain
+* Approving the 25 existing MakerDAO oracle feeds as signers for Wormhole.
+* Approving a quorum requirement of 13 oracle feeds to complete a fast withdrawal.
+
+**Common Wormhole Parameters**
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/polls/Maker Wormhole Deployment - April 4, 2022.md:35]: Yep

File: governance/polls/Maker Wormhole Deployment - April 4, 2022.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+categories:
+   - Bridge
+   - Technical
+   - Risk Parameters
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-03-28T16:00:00
+end_date: 2022-03-31T16:00:00
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/polls/Maker Wormhole Deployment - April 4, 2022.md:15]: yep

File: governance/polls/Maker Wormhole Deployment - April 4, 2022.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* Fast Withdrawal Fee: Zero DAI
+
+Please review the discussion [thread](https://forum.makerdao.com/t/request-to-poll-for-maker-wormhole-deployment-and-associated-risk-parameters/14209) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* The deployment of Maker Wormhole will be included in a future executive vote as the Protocol Engineering Core Unit's schedule allows.
+* It is expected that this executive vote will take place within 30 days of this poll passing, absent external factors.
+* If the Executive Vote passes, then Wormhole will become active in the Maker Protocol after the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay) has expired.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/polls/Maker Wormhole Deployment - April 4, 2022.md:53]: Copied this part from an old poll

File: governance/polls/PPG - Open Market Committee Proposal - April 4, 2022.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+**[Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Decreases**
+- Decrease the **WSTETH-A** Stability Fee from 2.5% to 2.25%.
+- Decrease the **CRVV1ETHSTETH-A** Stability Fee from 3.5% to 2.25%.
+- Decrease the **WBTC-A** Stability Fee from 3.75% to 3.25%.
+- Decrease the **WBTC-B** Stability Fee from 5% to 4.5%.
+- Decrease the **GUNIV3DAIUSDC1-A** Stability Fee from 0.1% to 0%.
+- Decrease the **GUNIV3DAIUSDC2-A** Stability Fee from 0.25% to 0.05%.
+
+**[Maxmimum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Increases**
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/polls/PPG - Open Market Committee Proposal - April 4, 2022.md:37]: goddamn it grammarly you're supposed to protect me from this BS.

File: governance/polls/PPG - Open Market Committee Proposal - April 4, 2022.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+**D3M [Target Borrow Rate](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar) Increase**
+- Increase the **DIRECT-AAVEV2-DAI** Target Borrow Rate from 2.85% to 3.5%.
+
+Please review the proposal [thread](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-31-march-2022/14347) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* These parameter changes will be included in the weekly Executive Vote on the Friday following the conclusion of this poll.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/polls/PPG - Open Market Committee Proposal - April 4, 2022.md:52]: MOMC ones are usually pretty easy. For now this is fine. When PE makes changes we'll just write everything as 'in the next month, I think'

File: governance/polls/PPG - Open Market Committee Proposal - April 4, 2022.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+---
+title: PPG - Open Market Committee Proposal - April 4, 2022
+summary: Small Stability Fee decreases on multiple vault types, Aave D3M Target Borrow Rate increase, Debt Ceiling increases on GUNIV3DAIUSDC vaults types, and other minor changes.
```
Comment (patrick-j-govalpha) [inline governance/polls/PPG - Open Market Committee Proposal - April 4, 2022.md:3]: "vaults types", sorry I didn't spot it first time round

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

**Review by patrick-j-govalpha — APPROVED** [review]

**Pull request merged**